lib/sidebar.js

```diff
@@ -21,12 +21,10 @@ define(function () {
     sidebar.appendChild(container);
 
     self.getWidth = function getWidth() {
-      if (sidebar.classList.contains('hidden')) {
+      if (sidebar.classList.contains('hidden') || button.offsetHeight === 0) {
```

Comment:
this should fully handle the breakpoint and no code :)
https://github.com/ffrgb/meshviewer/blob/develop/scss/modules/_variables.scss#L24